flake-module.nix

```diff
@@ -120,6 +120,15 @@ ihpFlake:
                     includeDevTools = false;
                     optimized = false;
                 };
+
+                migrate =
+                    let
+                        projectSrc = pkgs.nix-gitignore.gitignoreSource [] cfg.projectPath;
+                    in
+                        pkgs.writeScriptBin "migrate" ''
```

Comment:
As long as this package is build with `nix build` it shouldn't cause any collision. I'd prefer to not introduce two names for the same binary, otherwise it will be confusing to other people in the future 👍